graphql/schemabuilder/types.go

```diff
@@ -331,6 +336,10 @@ type method struct {
 	// FederationType is an object where all the fields are keys
 	// that can be exposed over federation
 	FederationType interface{}
```

Comment:
How about change this to RootObjectType and add a comment to say
"RootObjectType is the reflect type of parent object if it is a root object" ?

Reply:
The motivation is the word federation is kind of confusion to me.